Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1

```diff
@@ -1694,11 +1698,6 @@ https://psappdeploytoolkit.com
     ## Get the name of this function
     [String]${CmdletName} = $PSCmdlet.MyInvocation.MyCommand.Name
 
-    ## Stop the Close Program Dialog if running
-    If ($formCloseApps) {
-        $formCloseApps.Close
```

Comment:
Did you have a reason for removing this ?

Reply:
It might be best to read the PR descriptions for some of these. I enabled strict compliance mode (`Set-StrictMode -Version Latest`) and started to comb through the errors. This variable is not defined anywhere at all, therefore it's an access on an undefined variable and was causing a throw. It looks like something that's been left behind from a previous setup.